hxd/Charset.hx

```diff
@@ -138,6 +138,15 @@ class Charset {
 		return isSpace(code) || isCJK(code);
 	}
 
+	static var complementChars : Map<Int,Bool> = {
+		var str = "ヽヾーァィゥェォッャュョヮヵヶぁぃぅぇぉっゃゅょゎゕゖㇰㇱㇲㇳㇴㇵㇶㇷㇸㇹㇺㇻㇼㇽㇾㇿ々〻";
+		[for( i in 0...str.length ) str.charCodeAt(i) => true];
+	}
+
+	public function isComplementChar(code) {
+		return complementChars.exists(code);
+	}

```

Comment:
```suggestion
	static var complementChars : Array<Int> = {
		var str = "ヽヾーァィゥェォッャュョヮヵヶぁぃぅぇぉっゃゅょゎゕゖㇰㇱㇲㇳㇴㇵㇶㇷㇸㇹㇺㇻㇼㇽㇾㇿ々〻";
		[for ( i in 0...str.length ) str.charCodeAt(i)];
	}

	public function isComplementChar(code) {
		return complementChars.indexOf(code) != -1;
	}

```
There's no point in having a Map here, only increases memory footprint for no reason, since it contains only chars that would prevent line breaks and nothing else, especially since check only tests for value being here, not whether its `true` or `false`.

Reply:
I was the one suggesting Map , so we get a lot faster check, given it can gets called a lot.

Reply:
I suppose it's due to IntMap using different lookup approach? (something like binary tree?) I'm curious why Map is faster in that case.